docs/zpy/about.md

```diff
@@ -2,6 +2,7 @@
 
 `zpy` is a python package that makes synthetic data easy, by simplifying the process of creating simulations, or [sims](https://zumolabs.github.io/zpy/zpy/tutorials/what_is_a_sim/). The `zpy` module contains multiple pieces:
 
-- A [Blender Addon](https://zumolabs.github.io/zpy/addon/about/) for creating and debugging [sims](https://zumolabs.github.io/zpy/zpy/tutorials/what_is_a_sim/) inside the Blender UI.
-- A [CLI](https://zumolabs.github.io/zpy/cli/about/) for things like uploading [sims](https://zumolabs.github.io/zpy/zpy/tutorials/what_is_a_sim/) and generating datasets.
-- A [WebApp](https://zumolabs.github.io/zpy/app/about/) for a GUI version of the API and CLI.
\ No newline at end of file
+- A [Blender Addon](../../addon/about) for creating and debugging [sims](https://zumolabs.github.io/zpy/zpy/tutorials/what_is_a_sim/) inside the Blender UI.
+- A [CLI](../../cli/about/) for things like uploading [sims](https://zumolabs.github.io/zpy/zpy/tutorials/what_is_a_sim/) and generating datasets.
+- A [WebApp](../../app/about/) for a GUI version of the API and CLI.
+- A [Client API](../../client_api/about/) for generating synthetic datasets.
```

Comment:
Id probably rename this to "Python Client"

Reply:
Done